statsmodels/regression/special_linear_model.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov 13 16:24:12 2018
+
+Author: Josef Perktold
+"""
```

Comment:
Is there a more informative name than “special”?  (or is that standard in a literature I’m not familiar with?)

Reply:
There is not a literature common name for it.
This is just "linear models for special cases" (with limited functionality).
However, the intention is to have the models computationally optimized for those special cases.